config.go

```diff
@@ -34,7 +34,7 @@ var (
 
 	// ClientVersion is the commandline version string used to control updating
 	// the CLI. The format is the calendar date (YYYY-MM-DD).
-	ClientVersion = "2024-05-17"
+	ClientVersion = "2024-05-22a"
```

Comment:
nit: 2024-05-22 unless there's already been an update to the client on the 22nd.

Reply:
my other pr was on 22 but updated to 23 !